<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="SKY-20140314HVQ" name="cn.xuexibao.business.spider.multiwork.JYWorkTest" tests="1" failures="1" timestamp="7 May 2014 07:52:12 GMT" time="18.017" errors="0">
  <testcase name="testWork" time="18.017" classname="cn.xuexibao.business.spider.multiwork.JYWorkTest">
    <failure type="java.lang.Error" message="Unresolved compilation problem: 
	exitWhenComplete cannot be resolved to a variable
">
      <![CDATA[java.lang.Error: Unresolved compilation problem: 
	exitWhenComplete cannot be resolved to a variable

	at cn.xuexibao.business.spider.multiwork.MultiWork.waitThreadRelease(MultiWork.java:163)
	at cn.xuexibao.business.spider.multiwork.MultiWork.run(MultiWork.java:116)
	at cn.xuexibao.business.spider.multiwork.JYWorkTest.testWork(JYWorkTest.java:32)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:702)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:894)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1219)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:768)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:87)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1188)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1113)
	at org.testng.TestNG.run(TestNG.java:1025)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:109)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:202)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:173)
]]>
    </failure>
  </testcase> <!-- testWork -->
</testsuite> <!-- cn.xuexibao.business.spider.multiwork.JYWorkTest -->
